bears/csv/CSVLintBear.py

```diff
@@ -1,11 +1,33 @@
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
 @linter(executable='csvlint',
         output_format='regex',
-        output_regex=r'\d\. (?P<message>.+(s|g|e|w|d)\.*)'
-                     r'( |$)(?P<additional_info>.*)')
+        output_regex=r'\n[0-9]+\.\s(?P<message>(?P<severity>\w+))\.\s'
+                     r'Row:\s(?P<line>[0-9]+).+',
+        severity_map={
+                    "wrong_content_type": RESULT_SEVERITY.MAJOR,
+                    "ragged_rows": RESULT_SEVERITY.MAJOR,
+                    "blank_rows": RESULT_SEVERITY.MAJOR,
+                    "invalid_encoding": RESULT_SEVERITY.MAJOR,
+                    "not_found": RESULT_SEVERITY.MAJOR,
+                    "stray_quote": RESULT_SEVERITY.MAJOR,
+                    "unclosed_quote": RESULT_SEVERITY.MAJOR,
+                    "whitespace": RESULT_SEVERITY.MAJOR,
+                    "line_breaks": RESULT_SEVERITY.MAJOR,
+                    "no_encoding": RESULT_SEVERITY.NORMAL,
+                    "encoding": RESULT_SEVERITY.NORMAL,
+                    "no_content_type": RESULT_SEVERITY.NORMAL,
+                    "excel": RESULT_SEVERITY.NORMAL,
+                    "check_options": RESULT_SEVERITY.NORMAL,
+                    "inconsistent_values": RESULT_SEVERITY.NORMAL,
+                    "empty_column_name": RESULT_SEVERITY.NORMAL,
+                    "duplicate_column_name": RESULT_SEVERITY.NORMAL,
+                    "title_row": RESULT_SEVERITY.NORMAL,
+                    "nonrfc_line_breaks": RESULT_SEVERITY.INFO,
+                    "assumed_header": RESULT_SEVERITY.INFO})
```

Comment:
I don't think those should be treated as severities, rather as `<origin>` right?